js/apps/admin-ui/src/components/dynamic/UrlComponent.tsx

```diff
@@ -0,0 +1,30 @@
+import { FormGroup } from "@patternfly/react-core";
+import { useTranslation } from "react-i18next";
+
+import { HelpItem } from "ui-shared";
```

Comment:
This import is no longer called `ui-shared` which is causing the build to fail, it should be renamed to `@keycloak/keycloak-ui-shared`.
```suggestion
import { HelpItem } from "@keycloak/keycloak-ui-shared";
```